### PR TITLE
Migrate to new hoodie architecture

### PR DESCRIPTION
**Pull request by finnp**
Migrate to new architecture as part of https://github.com/hoodiehq/hoodie/issues/458

### CODE REVIEW

File: lib/get-api.js [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
 
 var internals = module.exports.internals = {}
-internals.Store = require('hoodie-client-store')
-internals.Account = require('hoodie-client-account')
-internals.ConnectionStatus = require('hoodie-client-connection-status')
-internals.Log = require('hoodie-client-log')
+internals.Store = require('@hoodie/store/client')
+internals.Account = require('@hoodie/account/client')
+internals.ConnectionStatus = require('@hoodie/connection-status')
+internals.Log = require('@hoodie/log/client')
```
Comment (gr2m) [inline lib/get-api.js]: let's do `@hoodie/connection-status/client` as well

**Comment by gr2m** [comment]
besides the one comment, LGTM :)

**Comment by HipsterBrown** [comment]
Great work once again @finnp! LGTM once @gr2m's comment has been addressed. :)

**Comment by ashanaweb** [comment]
module.exports.internals = {}
-internals.Store = require('hoodie-client-store')
-internals.Account = require('hoodie-client-account')
-internals.ConnectionStatus = require('hoodie-client-connection-status')
-internals.Log = require('hoodie-client-log')
+internals.Store = require('@hoodie/store/client')
+internals.Account = require('@hoodie/account/client

**Comment by gr2m** [comment]
@ashanaweb what are you doing here, exactly?

**Comment by ashanaweb** [comment]
client-account), [store](https://github.com/hoodiehq/hoodie-client-store) and [task](https://github.com/hoodiehq/hoodie-client-task)
 +Hoodie’s client glue code that integrates Hoodie’s client core modules: [account](https://github.com/hoodiehq/hoodie-account-client), [store](https://github.com/hoodiehq/hoodie-store) and [task](https://github.com/hoodiehq/hoodie-client-task)

**Comment by ashanaweb** [comment]
client-account), [store](https://github.com/hoodiehq/hoodie-client-store)
and [task](https://github.com/hoodiehq/hoodie-client-task)
+Hoodie’s client glue code that integrates Hoodie’s client core modules: [
account](https://github.com/hoodiehq/hoodie-account-client), [store](
https://github.com/hoodiehq/hoodie-store) and [task](
https://github.com/hoodiehq/hoodie-client-task)

2016-04-02 20:32 GMT+02:00 Gregor Martynus <notifications@github.com>:

**Comment by finnp** [comment]
Thanks for the feedback. I updated the PR.

**Pull request merged**